### DISCUSSION TITLE
Welcome to ZenStates Discussions!

### DISCUSSION THREAD

**Discussion by irusanov** [irusanov/ZenStates#18] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by moltycat** [comment]
Hello. Can you made different profiles for manual setting on hotkey to new version of ZenState? It would be very usefull to have ability to switch profiles between silent and prformance for example.

**Reply by irusanov** [reply]
Yes, that's in the plans. Also some equivalent of Asus Dynamic OC.
The problem is I don't have time for this project at the moment and don't know for how long.

**Reply by moltycat** [reply]
Well, I will wait as long as it takes) There are no analogues of your program, only with your program I was able to configure the autoapply of undervolting on my laptop. Thank you for zenstates

**Comment by Mido-007** [comment]
**Hi .. Does ZenStates works fine without problems in ryzen 5 3600X CPU and with B450 Aorus Elite motherboard ??**

**Reply by moltycat** [reply]
Hi. I have clevo laptop with r5 3600 and b450 motherboard. For me it works fine.
